CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
   additional check is performed to see whether the token has been revoked. 
 - [added] A new method ['FirebaseAuth.revokeRefreshTokens(uid)'](https://firebase.google.com/docs/reference/admin/java/reference/com/google/firebase/auth/FirebaseAuth.html#revokeRefreshTokens)
```

Comment:
revokeRefreshTokensAsync()

Also mind the quotes: backtick instead of single quote.